### PR TITLE
fixed test failures and dependency

### PR DESCRIPTION
**Pull request by shashank-google**
fixed failing test
updated a dependency

### CODE REVIEW

**Comment by shashank-google** [comment]
@hhasija can you please review the same

File: java/pom.xml [hunk 1]
```diff
@@ -127,7 +127,7 @@
     <dependency>
       <groupId>com.google.oauth-client</groupId>
       <artifactId>google-oauth-client</artifactId>
-      <version>1.32.1</version>
+      <version>1.33.3</version>
```
Comment (ajayydv) [inline java/pom.xml:130]: shall we make this as maven variable?

File: java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java [hunk 2]
```diff
@@ -72,45 +72,9 @@ public GCStoBigquery() {
...
-        GCS_OUTPUT_DATASET_NAME,
-        bigQueryDataset,
-        GCS_OUTPUT_TABLE_NAME,
-        bigQueryTable,
-        GCS_BQ_INPUT_FORMAT,
-        inputFileFormat,
-        GCS_BQ_LD_TEMP_BUCKET_NAME,
-        bqTempBucket);
-
     LOGGER.info("input format: " + inputFileFormat);
```
Comment (ajayydv) [inline java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java:78]: Lets reformat this for lazy initialization in logger. i.e
LOGGER.info("input format: {}", inputFileFormat);

File: java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java [hunk 3]
```diff
@@ -160,4 +124,44 @@ public void runTemplate() {
       }
     }
   }
+
```
Comment (ajayydv) [inline java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java:127]: Please add a javadoc comment for function.

File: java/src/main/java/com/google/cloud/dataproc/templates/hive/HiveToGCS.java [hunk 4]
```diff
@@ -122,4 +94,35 @@ public void runTemplate() {
     LOGGER.info("HiveToGcs job completed.");
     spark.stop();
   }
+
+  void validateInput() {
```
Comment (ajayydv) [inline java/src/main/java/com/google/cloud/dataproc/templates/hive/HiveToGCS.java:98]: Shall we consider moving validateInput to BaseTemplate? I understand that input parameters are different for each parameter but its worth consolidating them in base class via var args or lambada function. We can create a seperate jira for this if required.

**Review by ajayydv — CHANGES_REQUESTED** [review]
Thanks for working on this Shashank. Added few comments. LGTM with those addressed.

File: java/src/main/java/com/google/cloud/dataproc/templates/hive/HiveToGCS.java (hunk 4 above)
Comment (shashank-google) [reply java/src/main/java/com/google/cloud/dataproc/templates/hive/HiveToGCS.java:98]: yeah we should create a separate git issue for code refactoring.

File: java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java (hunk 2 above)
Comment (shashank-google) [reply java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoBigquery.java:78]: Fixing it. FYI, it is not a change made in this pull request.

**Pull request merged**